examples/deploy_ecs/tests/test_defs.py

```diff
@@ -0,0 +1,11 @@
+from ..defs import defs, my_job
+
+
+def test_defs():
+    # The job name corresponds to the name of the function defining the job.
+    # In our example, the job is created from a graph,
+    # which is defined by the function `my_graph()`,
+    # therefore the name of the job is `my_graph`.
```

Comment:
then you wouldn't need this bit of weirdness

Reply:
Much better, done in [b1e28ca](https://github.com/dagster-io/dagster/pull/19038/commits/b1e28ca8a287e5719ac9807920d88001bc40de20)